content/en/guides/hosting/iam/authentication/service-accounts.md

```diff
@@ -4,19 +4,35 @@ displayed_sidebar: default
 title: Use service accounts to automate workflows
 ---
 
-A service account represents a non-human or machine user that can automatically perform common tasks across projects within a team or across teams. 
+A service account represents a non-human or machine user that can automatically perform common tasks across projects within a team or across teams. Service accounts are ideal for CI/CD pipelines, automated training jobs, and other machine-to-machine workflows.
 
-- An org admin can create a service account at the scope of the organization.
-- A team admin can create a service account at the scope of that team.
-	
-A service account's API key allows the caller to read from or write to projects within the service account's scope.
+## Key benefits
+
+- **No license consumption**: Service accounts do not consume user seats or licenses
+- **Dedicated API keys**: Secure, non-expiring credentials for automated workflows
+- **Flexible scoping**: Available at both organization and team levels
+- **User attribution**: Optionally attribute automated runs to human users
```

Comment:
Maybe explain here that the service account operates on behalf of the user that owns it.